src/layouts/sidebar-sidecar/index.tsx

```diff
@@ -71,6 +71,14 @@ const SidebarSidecarLayoutContent = ({
 		offset: [`${stickyNavHeaderHeight * -1}px start`, `end end`],
 	})
 
+	useEffect(() => {
```

Comment:
I understand that this works now, because we only use main on docs/tutorials etc. But as an aside we should _really_ be using main on all of our pages. e.g. Homepage, Certs, Etc.

Reply:
Migrating to that standard is beyond the scope of this PR.